index.html

```diff
@@ -0,0 +1,130 @@
+                <a class="nav-link" href="instagram.html">Gram</a>
+              </li>
+          </ul>
+        </div>
+      </nav>
+
+
+
+<!-- Header-->
+<header class ="header">
+  <div class="overlay">
+    <div class="container">
+      
+      <div class="description text-left">
+  <h1> Kizomba House of Austin, TX</h1>
+  <p>  Lorem ipsum is placeholder text commonly used in the graphic, print, and publishing industries for previewing layouts and visual mockups.</p>
+     </div>
+        </div>
+</div>
+    
+</header>
+
+
+    
+<!-- About Community Section -->
+<div class="community">
+  <div class="container">
+    <div class="row">
+
+      <div class="col-lg-8 col-md-6 col-sm-12">
+        <div class ="text text-center">
+        <h1>About ATX Kiz Community</h1>
+        <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Cursus euismod quis viverra nibh cras pulvinar mattis. Dui accumsan sit amet nulla facilisi morbi tempus iaculis urna. Eu feugiat pretium nibh ipsum. Sit amet justo donec enim diam vulputate ut pharetra sit. Tellus elementum sagittis vitae et leo duis ut. Morbi tristique senectus et netus et malesuada fames. Sit amet nulla facilisi morbi tempus iaculis urna id. Sollicitudin nibh sit amet commodo nulla facilisi nullam.</p>
+            </div>
+      </div>
+
+      <div class="col-lg-4 col-md-6 col-sm-12">
+        <div class="buttons">
+        <a href="http://" class="button">Events</a>
+        <a href="http://" class="button">Music</a>
+        <a href="http://" class="button">Classes</a>
+        <a href="http://" class="button">Donations</a>
+      </div>
+</div>
+    </div>
+  </div>
+  </div>
+
+
+
+
+<!-- About Kizomba Section -->
+<div class="kizomba">
+<div class="container">
+  <div class="text text-center">
+  <h1>What is Kizomba?</h1>
+  <p>  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Cursus euismod quis viverra nibh cras pulvinar mattis. Dui accumsan sit amet nulla facilisi morbi tempus iaculis urna. Eu feugiat pretium nibh ipsum. Sit amet justo donec enim diam vulputate ut pharetra sit. Tellus elementum sagittis vitae et leo duis ut. Morbi tristique senectus et netus et malesuada fames. Sit amet nulla facilisi morbi tempus iaculis urna id. Sollicitudin nibh sit amet commodo nulla facilisi nullam.</p>
+</div>
+</div>
+
+<!--Carousele-->
+<div id="carouselExampleControls" class="carousel slide" data-ride="carousel">
+  <div class="carousel-inner">
+    <div class="carousel-item active">
+      <img class="img" src="images/kizparty1.png" class="d-block w-100" alt="...">
+    </div>
+    <div class="carousel-item">
+      <img class="img" src="images/kizparty2.png" class="d-block w-100" alt="...">
+    </div>
+    <div class="carousel-item">
+      <img class="img" src="images/kizparty3.png" class="d-block w-100" alt="...">
```

Comment:
I assume you'll add a better `alt="..."` later? If it's not important, just leave an empty `alt=""`

Reply:
Yes I will change pictures as well.
Any chance you know how I can connect the carousel to instagram images?
Idk if it is possible

Reply:
What do you mean by that? Do you mean a dynamic up-to-date connection to an Instagram account? Hum... I confess I never did it. You'll need to look for "Instagram integration on websites".